benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/configuration/goal/GoalTest.scala

```diff
@@ -14,7 +14,17 @@ import scala.util.Try
 class GoalTest extends JUnitSuite {
 
   private val completeGoalYaml: String =
```

Comment:
It is done an another test: https://github.com/benchflow/benchflow/blob/feature-dsl-all-definition-sections/benchflow-dsl/src/test/scala/cloud/benchflow/dsl/BenchFlowDSLTest.scala#L63